// Proto file representing the EDB Term class

syntax = "proto3";

package ansys.api.edb.v1;

import "edb_messages.proto";

service TerminalService {
  // Sets a reference on point term
  rpc SetReference(TermSetRefMessage) returns (google.protobuf.BoolValue) {}
}

message TermSetRefMessage {
  EDBObjMessage source_term = 1;
  EDBObjMessage ref_term = 2;
}
